### PR TITLE
Fixed minor typo

### PR DESCRIPTION
**Pull request by JSchwehn**
I assume there was a h missing, so I added it.

### CODE REVIEW

File: tt.py [hunk 1]
```diff
@@ -278,4 +278,4 @@ def main():
 
 
 # Starting here
-main()
\ No newline at end of file
+main()
```
Comment (phako) [inline tt.py:281]: Looks unrelated...

**Comment by fallenbeck** [comment]
Thanks
